Admin/AbstractAdmin.php

```diff
@@ -2838,22 +2838,12 @@ public function getDashboardActions()
     }
 
     /**
-     * Setting to true will enable mosaic button for
-     * the admin screen.
-     * Setting to false will hide mosaic button for
+     * Setting to true will enable mosaic button for the admin screen. Setting to false will hide mosaic button for
      * the admin screen.
      */
-    public function setIsShowMosaicButton($isShow)
+    public function setIsShowMosaicButton($isShown)
```

Comment:
> If somebody wants to hide mosaic button only if some condition equal true, in ListMapper - he can call method hideMosaicButton. I see in this solution two advantages:
1. We hide realization of method and listModes variable name.
2. IDE can highlight this method unlike unset($this->listModes['mosaic'])

Well he could also call `showMosaicButton($oppositeCondition)`, which also has advantages 1 and 2. The admin class is already quite huge, and your PR adds 4 methods…